lib/useScoresOfGolfer.js

```diff
@@ -0,0 +1,33 @@
+import useSWR from 'swr'
+import { getToken } from './userAuth'
+
+export const GOLFERS_URL = `${process.env.NEXT_PUBLIC_API_URL}/golfers`
+
+const useScoresOfGolfer = id => {
+  const fetcher = async url => {
+    const res = await fetch(url, {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${getToken()}`,
+      },
+    })
+
+    if (!res.ok) {
+      const error = new Error('An error occurred while fetching the data.')
+      error.info = await res.json()
+      error.status = res.status
+      throw error
+    }
+    return res.json().then(data => data.scores)
+  }
+  const GOLFERS_URL_WITH_ID = GOLFERS_URL + '/' + id
```

Comment:
Let's not use another constant for the URL. Try to create a function that returns a URL for a given id.

Example:

```
export const GOLFERS_URL = id = `${process.env.NEXT_PUBLIC_API_URL}/golfers/#{id}`
```